### ISSUE TITLE
[Issue] Date of publishing of the video, missing from thumbnail small row

### ISSUE THREAD

**Issue by costinmx** [yuliskov/SmartTubeNext#1497]
Since 4-5 days, date of published video is randolmly missing from most of the thumbnails second row, for most channels.
Instead we get useless info like "Song", "New" etc. 
'Date of publish' is a must, next to 'channel name' and 'view count'. Establishing how old is a video is now impossible without playing. Not all videos, not all channels have this problem. But most, do.
And looks like original YT has similar problem, maybe there is a fix on STN end also, from our brave @yuliskov

**Comment by costinmx** [comment]
Just checked, NewPipe looks not to be affected by this issue. All thumbnails displayed have the date info (how long ago time) on small row, regardless of YT/user setting. So it could be a developer choice.

**Comment by yuliskov** [comment]
How to reproduce? I need open a channel of any music artist, right?

Regards, Yuriy

**Comment by costinmx** [comment]
Yes. Take this three channels as example: DJanes Net, Eugene Belsky and Radio Intense. 
Just did some more digging into this issue: 
Some clips are marked as "Song" - those clips will no longer display "publishing time" in thumbnail small row in STN and YT app. But they will display normally into NewPipe and Browser YT website.
Screenshots attached from YT, STN and NP.  I wonder if  this can be sorted out from YT API. All hopes and prayers go to you.

![Screenshot_20220512-000226](https://user-images.githubusercontent.com/74626232/167960724-76297d37-f115-4390-952b-52b16c59e1be.png)
![Screenshot_20220512-000247](https://user-images.githubusercontent.com/74626232/167960749-ec634ee3-ec4c-4400-8149-8e3a89213fda.png)
![Screenshot_20220512-002700](https://user-images.githubusercontent.com/74626232/167960813-73d66304-2b9c-4714-865
![Screenshot_202
![Screenshot_20220512-002246](https://user-images.githubusercontent.com/74626232/167960918-43d4f46f-d724-4ffa-a789-347d8be8959c.png)
20512-001734](https://user-images.githubusercontent.com/74626232/167960900-45b04665-f024-4c15-83b3-9cbf63dcf31d.png)
9-1
![Screenshot_20220512-002410](https://user-images.githubusercontent.com/74626232/167960840-7863a640-b72f-4a9b-a11f-0ada2092c039.png)
30db81898d0.png)
![Screenshot_20220512-002800](https://user-images.githubusercontent.com/74626232/167960945-199667df-ee50-45aa-9a67-a59afcfe6fca.png)

**Comment by yuliskov** [comment]
@costinmx thx for the help. Seems it's my api limitation (I'm using tv api mainly). But I'll try find a a solution. Maybe I'll switch to the browser api instead.

**Comment by costinmx** [comment]
Glad to help in any way, it's a least I can do. Defintely looks like something changed at YT. Maybe, because of YT Music. Just hope that an eventual change of API in STNext, does not also remove other useful features.
I really appreciate all the hard work you put into this project, and I am asking, pretty please avoid removing previous useful features (like Add Remove from recent playlist)... 
Regards

**Comment by netahoy** [comment]
Yes having the published date/time helps in identifying current videos and in general too.

As a workaround till the actual date can be incorporated within the thumbnail, can we have those details on the Video description popup?

Thanks

**Comment by y-lee** [comment]
Is this related to the longstanding bug regarding not showing the date of the videos (and the how-much-of-video-is-watched indicator) in the Suggestions page at the end of a video?

**Comment by yuliskov** [comment]
@y-lee correct. You're in the right place.

**Comment by costinmx** [comment]
To me it looks like a Youtube "new feature". Every single video marked as 'Song' is not showing the age of the video in thumbnail anymore. Thats a fact. As Yuriy concluded some time ago: This is valid for all apps made with Video API, like STN or YT official app. Web API is unaffected, yet (like YT Webpage and NP app). I think there's no way out until YT fixes it.
